### PR TITLE
refactor: update sdk and api url

### PR DESCRIPTION
**Pull request by 0xsambugs**
In here we are updating to the latest version of the SDK and we are also using the new API endpoint

### CODE REVIEW

File: src/services/sdkService.ts [hunk 1]
```diff
@@ -37,17 +37,17 @@ export default class SdkService {
...
                 list: {
                   type: 'api',
-                  baseUri: ({ chainId }: { chainId: number }) =>
-                    `https://api.mean.finance/v1/swap/networks/${chainId}/quotes`,
+                  baseUri: ({ chainId, sourceId }: { chainId: number; sourceId: SourceId }) =>
+                    `https://api.mean.finance/v1/swap/networks/${chainId}/quotes/${sourceId}`,
                   sources: SOURCES_METADATA,
                 },
-                sourceIds: ['uniswap', 'odos', 'rango', '0x', 'firebird'],
+                sourceIds: ['uniswap', 'odos', 'rango', '0x', 'firebird', 'changelly'],
```
Comment (0xsambugs) [inline src/services/sdkService.ts:50]: We are adding changelly!

File: src/services/sdkService.ts [hunk 2]
```diff
@@ -28,7 +28,7 @@ export default class SdkService {
     this.sdk = buildSDK({
       provider: {
         source: {
-          type: 'only-first-provider-that-supports-chain',
+          type: 'prioritized',
```
Comment (0xsambugs) [inline src/services/sdkService.ts:31]: Just renamed

File: src/services/sdkService.ts [hunk 3]
```diff
@@ -37,17 +37,17 @@ export default class SdkService {
           type: 'overridable-source-list',
           lists: {
             default: {
-              type: 'default',
+              type: 'local',
```
Comment (0xsambugs) [inline src/services/sdkService.ts:40]: Another rename

File: src/services/sdkService.ts [hunk 4]
```diff
@@ -206,9 +205,13 @@ export default class SdkService {
       return this.walletService.getCustomToken(address);
     }
 
-    const tokenResponse = await this.sdk.tokenService.getTokens({ addresses: [{ addresses: [address], chainId }] });
+    const tokenResponse = await this.sdk.metadataService.getMetadataForChain({
```
Comment (0xsambugs) [inline src/services/sdkService.ts:208]: We can now fetch the name!! 🎉 

I marked it as required, is that ok? Other options would be "best effort" or "can ignore"

**Review by FiboApe — APPROVED** [review]

**Pull request merged**